src/components/Semester.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <!-- stolen from this example: https://vuetifyjs.com/en/components/cards#grids -->
   <v-expansion-panel-content
+    v-if="hideIap ? semesterType !== 'IAP' : true"
```

Comment:
Nitpick, but (!hideIAP || semesterType !== 'IAP') is clearer than the ternary imo.